cg/cli/upload/scout.py

```diff
@@ -252,7 +239,7 @@ def upload_multiqc_to_scout(context: CGConfig, case_id: str, dry_run: bool) -> N
 def get_upload_api(case: Family, cg_config: CGConfig) -> UploadAPI:
     """Return the upload API based on the data analysis type"""
 
-    analysis_apis: Dict[Pipeline, UploadAPI] = {
+    analysis_apis: dict = {
```

Comment:
See the comment I made for @Vince-janv But since you both agree do you still think this is a good idea? I agree that it was better before to the eye but it assumes that you also know the metaclass of the different upload-apis